src/seedu/addressbook/commands/Command.java

```diff
@@ -39,7 +39,7 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     /**
      * Executes the command and returns the result.
      */
-    public abstract CommandResult execute();
+    public abstract boolean isMutating();
```

Comment:
Missing/wrong header comment? All non-trivial classes/ methods should have javadoc format header comments.